web/src/components/vis/ScorePlot.vue

```diff
@@ -342,14 +330,14 @@ export default {
           .attr('cy', 0)
           .attr('r', 1)
           .attr('style', 'fill: none; stroke: black;')
```

Comment:
This `stroke:black` on the `<circle>` is overriding the `stroke` color on the parent `<g>` element in the update phase, so for me all circles are black. Changing to
```
.style('fill', 'none')
```
seems to fix it.